source/Generic/PlayState/PlayState.cs

```diff
@@ -591,6 +594,22 @@ private void RemoveGameStopwatchTuple(Game game)
             }
         }
 
+        private void InitializePlaytimeInfoFile() // Temporary workaround for sharing PlayState paused time until Playnite allows to share data among extensions
+        {
+            // This method will remove the info of the txt file in order to avoid reusing the previous play information.
+            string[] info = { " ", " " };
+
+            File.WriteAllLines(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Playnite", "ExtensionsData", "PlayState.txt"), info);
```

Comment:
Please use the `ExtensionsDataPath` path from the Paths API and not environment variables https://playnite.link/docs/api/Playnite.SDK.IPlaynitePathsAPI.html#Playnite_SDK_IPlaynitePathsAPI_ExtensionsDataPath

Should be `PlayniteApi.Paths.ExtensionsDataPath`. You can use the `ConfigurationPath` if you prefer also but preferably `ExtensionsDataPath`

Reply:
I will check it, thanks!